ui/app/adapters/application.js

```diff
@@ -36,7 +36,7 @@ export default RESTAdapter.extend({
     return false;
   },
 
-  addHeaders(url, options) {
+  addHeaders(url, options, method) {
```

Comment:
need to confirm adding this header is accurate - this change is based on and API error returned from the backend when attempting to send a `PATCH` request

Reply:
confirmed! Received same error with the latest binary
<img width="1102" alt="Screenshot 2023-12-12 at 11 45 16 AM" src="https://github.com/hashicorp/vault/assets/68122737/705cfbf0-60cf-4823-be6a-be75ab17e5a5">